modules/aws/master/variables.tf

```diff
@@ -104,7 +104,7 @@ variable "volume_size_etcd" {
 
 variable "volume_size_root" {
   type    = string
-  default = 8
+  default = 30
```

Comment:
Won't it replace existing EBS volumes?

Reply:
yes it will, but all the volumes except fro etcd are ephemeral and any  machine roll will create a new disk so its not an issue